docs/application/dotnet/guides/applications/application_lifecycle.md

```diff
@@ -0,0 +1,183 @@
+            base.OnCreate();
+            InitSensors();
+            InitUI();
+        }
+    }
+```
+
+The following code snippet is responsible for initializing `Accelerometer` sensor. The function setup the event handler for data update and set the update interval for one second. After setup, the `SensorAccelerometer.Start()` to start measurements:
+
+```csharp
+    void InitSensors()
+    {
+        SensorAccelerometer = new Accelerometer();
+        SensorAccelerometer.Interval = 1000;
+        SensorAccelerometer.DataUpdated += OnAccelerometer;
+        SensorAccelerometer.Start();
+    }
+```
+
+The UI setup is based on the basic template. A single `TextLabel` component is used to show the application UI and its running state:
+
+```csharp
+    void InitUI()
+    {
+        Window.Instance.KeyEvent += OnKeyEvent;
+
+        TextLabel test = new TextLabel("Lifecycle test...")
+        {
+            HorizontalAlignment = HorizontalAlignment.Center,
+            VerticalAlignment = VerticalAlignment.Center,
+            TextColor = Color.Blue,
+            PointSize = 12.0f,
+            HeightResizePolicy = ResizePolicyType.FillToParent,
+            WidthResizePolicy = ResizePolicyType.FillToParent
+        };
+
+        Window.Instance.GetDefaultLayer().Add(test);
+    }
+```
+
+The `OnTerminate()` method logs information about the application termination:
+
+```csharp
+    protected override void OnTerminate()
+    {
+        Tizen.Log.Info("LifecycleTest", "OnTerminate()");
+        base.OnTerminate();
+    }
+```
+
+The `OnPause()` method is called when application is suspended, but not killed. Then, it goes into the `Paused` state. To stop reading data from the `Accelerometer`, the `SensorAccelerometer.Stop()` method is used. Now, the application reduces resource usage. The main application window goes into the background now. The UI events are not received, but the data event handler have to be stopped manually:
+
+```csharp
+    protected override void OnPause()
+    {
+        Tizen.Log.Info("LifecycleTest", "OnPause()");
+        base.OnPause();
+
+        if (SensorAccelerometer != null) SensorAccelerometer.Stop();
+    }
+```
+
+The Tizen operating system may resume `Lifecycle` application, if it was not terminated. The data updating is restored using the `SensorAccelerometer.Start()` method. The application main window should be visible now:
+
+```csharp
+    protected override void OnResume()
+    {
+        Tizen.Log.Info("LifecycleTest", "OnResume()");
+        base.OnResume();
+
+        if (SensorAccelerometer != null) SensorAccelerometer.Start();
+    }
+```
+
+The callback setup in `InitSensors()` function is called every second. It prints debug information about a measured data from `Accelerometer` sensor. In the emulator case, this value can be changed using `Control Panel` tool. For details, see [Log Viewer](/application/native/guides/error/system-logs.md#dlogutil):
```

Comment:
Could you please rephrase the reference text (For details, see .....) for "log viewer" inline with reference text in line #19?